src/app/widgets/weather/weather-admin/weather-admin.component.css

```diff
@@ -0,0 +1,96 @@
+.weather-admin {
```

Comment:
As I already wrote - can be done using utility classes - but it is just a convention.